docs/book.toml

```diff
@@ -43,3 +43,18 @@ site-url = "/bevy_cli/"
 "/bevy_lint/lints/suspicious/insert_event_resource/index.html" = "../../../../api/bevy_lint/lints/suspicious/insert_event_resource/index.html"
 "/bevy_lint/lints/suspicious/insert_unit_bundle/index.html" = "../../../../api/bevy_lint/lints/suspicious/insert_unit_bundle/index.html"
 "/bevy_lint/lints/suspicious/iter_current_update_events/index.html" = "../../../../api/bevy_lint/lints/suspicious/iter_current_update_events/index.html"
+
+
+# `mdbook-linkcheck2` <https://github.com/marxin/mdbook-linkcheck2> is a backend that
+# verifies links are correct.
+[output.linkcheck2]
+# You don't need `linkcheck` installed in order to build the website.
+optional = true
+# The API docs are built by `rustdoc` and merged separately, so `linkcheck` doesn't know they
+# exist.
+exclude = ["/api/bevy_lint"]
+# When enabled, this will send requests to websites to verify the pages exist. This is slow and
+# places a burden on those websites when checked constantly, so this should only be enabled
+# manually.
+follow-web-links = false
+user-agent = "mdbook-linkcheck/0.7.7 (https://github.com/TheBevyFlock/bevy_cli)"
```

Comment:
```suggestion
user-agent = "mdbook-linkcheck2 (https://github.com/TheBevyFlock/bevy_cli)"
```

(Nit) I think we should update it to say `mdbook-linkcheck2`, but I also think we should remove the version so we don't have to worry about it getting outdated.